man/tpm2_unseal.1.md

```diff
@@ -38,15 +38,22 @@ alive and pass that session using the **--input-session-handle** option.
 
     Output file name, containing the unsealed data. Defaults to stdout if not specified.
 
-  * **-S**, **--input-session-handle**=_SESSION\_HANDLE_:
+## Session Options
 
-    Optional Input session handle from a policy session for authorization.
+  Options used for controlling sessions and policy events.
+
+  * **-S**, **--session**=_SESSION\_FILE_:
+
+    Optional, A session file from **tpm2_startauthsession**(1)'s **-S** option. This session
+    is used in leui of starting a session a session and using the PCR policy options. **-L** is
+    mutually exclusive of this option.
 
   * **-L**, **--set-list**==_PCR\_SELECTION\_LIST_:
 
     The list of pcr banks and selected PCRs' ids.
     _PCR\_SELECTION\_LIST_ values should follow the
     pcr bank specifiers standards, see section "PCR Bank Specfiers".
+    **-S** is mutually exclusive of this option.
```

Comment:
I think it should be: mutually exclusive *with* this option

Reply:
Can you please add an example to the man page using this new option and also extend the unseal integration test to test for regressions?

Reply:
We have it in the tcti/abrmd tests since this abrmd specific. So I didn't want to jam it into the generic test suite in case users were running against in-kernel RM.

Reply:
I'm leaving *to*, as *with* just seems awkward when reading it.